ontology/yaml/resources/LIGHTING/entity_types/ABSTRACT.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the License);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an AS IS BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### ABSTRACT TYPES
+
+SS:
+  description: "Basic combination of run command and status (start/stop); indicates the light is active or inactive."
+  is_abstract: true
+  implements:
+  - /SS
+
+DC:
+  description: "Brightness dimming control."
+  is_abstract: true
+  uses:
+  - brightness_percentage_command
+
+MS:
+  description: "Motion sensing monitoring."
+  is_abstract: true
+  uses:
+  - motion_status
+
+OS:
+  is_abstract: true
+  description: "Occupancy status for a specific zone."
+  uses:
+  - zone_occupancy_status
+
+SC:
+  is_abstract: true
+  description: "Scene control for a device or group of devices."
+  uses:
+  - scene_command
```

Comment:
Didn't we decide this was scene index command?

Reply:
sure? i dont remember, but ive adjusted it.